cms/templates/howitworks.html

```diff
@@ -14,8 +16,8 @@
 <div class="wrapper-content-header wrapper">
   <section class="content content-header">
     <header>
-      <h1><span class="wrapper-text-welcome">${_("Welcome to {studio_name}").format(
-        studio_name=u'</span><span class="logo">{studio_name}</span>'.format(
+      <h1><span class="wrapper-text-welcome">${Text(_("Welcome to {studio_name}")).format(
+        studio_name=HTML(u'</span><span class="logo">{studio_name}</span>').format(
```

Comment:
@

I have created the sandbox https://studio-safetemplates12042.sandbox.edx.org/. If you are not logged in then you can access the page.

You would see, `Welcome to edX` headline. That is one under question.

@robrap is the PR ready to merge after @cptvitamin 's look/suggestion ?